bindings/web5_uniffi_wrapper/src/errors.rs

```diff
@@ -2,7 +2,7 @@ use serde_json::Error as SerdeJsonError;
 use std::fmt::Debug;
 use thiserror::Error;
 use web5::credentials::presentation_definition::PexError;
-use web5::credentials::CredentialError;
+use web5::credentials::VerificationError;
```

Comment:
i changed the name so vp and vc verification can share the same errors

Reply:
initial reaction is skepticism, but after more consideration I agree